tests/conftest.py

```diff
@@ -176,6 +179,10 @@ async def flush_default_collection(
     setup_auth: Auth,
 ):
     yield
+    # TODO: we should still set "setup_auth" to delete collection? So, edit
+    # readme to suggest setting SETUP_AUTH whether the tests are being run on
+    # DEV or locally? Or could EDITOR_AUTH be used here? Who is allowed to
+    # delete these records?
```

Comment:
SETUP_AUTH has admin perms. It can create and delete collections.

EDITOR_AUTH is a remote settings user.

For LOCAL, we setup the server. 
On DEV we shouldn't do any operation that require admin perms, and SETUP_AUTH can be omitted

Reply:
> On DEV we shouldn't do any operation that require admin perms, and SETUP_AUTH can be omitted

But do we still want to "clean up" after ourselves on DEV? Otherwise, we'll accumulate more and more records after every integration test. 

I think the `EDITOR_AUTH` user creates all of the records in our integration tests -- can they also delete their own records? If so, maybe we can use their credentials for `flush_default_collection`.

Reply:
> I think the EDITOR_AUTH user creates all of the records in our integration tests -- can they also delete their own records? If so, maybe we can use their credentials for flush_default_collection.

Indeed, an editor can delete their own records. But since editor is among the users that have write permissions on the `product-integrity` collection, they can also delete every records in it.